packages/salesforcedx-vscode-core/src/conflict/index.ts

```diff
@@ -15,4 +20,51 @@ export {
   DirectoryDiffer,
   DirectoryDiffResults
 } from './directoryDiffer';
+export const conflictView = ConflictView.getInstance();
 export const conflictDetector = ConflictDetector.getInstance();
+
+export async function setupConflictView(
+  extensionContext: vscode.ExtensionContext
+): Promise<void> {
+  const view = conflictView;
+  await view.init(extensionContext);
+}
+
+export async function registerConflictView(): Promise<vscode.Disposable> {
+  const viewItems: vscode.Disposable[] = [];
+
+  viewItems.push(
+    vscode.commands.registerCommand('sfdx.force.conflict.diff', entry =>
+      conflictDiff(entry)
+    )
+  );
+
+  viewItems.push(
```

Comment:
I found this a bit confusing, clicking on the name of the file opens the conflict but clicking the icon exclusively opens the file. Shouldn't we always just open the conflict if the user is clicking the conflict view ?

Reply:
I found this to be a useful capability. It allows direct navigation to just the project file. This is also how the VCS view works, so there is precedent for it.

Reply:
Got it, thanks for the context. Let's keep it as it is.